testsuite/MDAnalysisTests/coordinates/test_crd.py

```diff
@@ -89,23 +82,23 @@ class TestCRDWriterMissingAttrs(object):
                  'tempfactors': 0.0,
                  }
 
-    def _check_warns(self, missing_attr):
+    @pytest.mark.parametrize('missing_attr', req_attrs)
```

Comment:
The tests are failing because parametrizing with a dict causes an unpredictable order for xdist